src/content/blog/drupalcon-barcelona-2024.md

```diff
@@ -18,7 +18,7 @@ I spent the last week in Barcelona, Spain for DrupalCon Barcelona. It was so gre
 
 A cool thing about DrupalCon is that anybody can set up a mini-session, and no real approval process is needed. I typically propose several of these, since they're a great way to deal with niche topics with a few people and have nice, smaller conversations with people who have specific needs. This time I did:
 
-- **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://ddev.readthedocs.io/en/latest/users/install/ddev-installation/#ddev-installation-codespaces).
+- **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://docs.ddev.com/en/latest/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://docs.ddev.com/en/latest/users/install/ddev-installation/#ddev-installation-codespaces).
```

Comment:
```suggestion
- **Working together: DDEV on Gitpod and GitHub Codespaces**: We talked about the many ways to use DDEV in Cloud IDEs like Gitpod and GitHub Codespaces. People were really interested in this, since of course these approaches don't require a local development environment at all (or rather, the local environment is in the cloud on a VM, and presented via a browser). See the DDEV docs on [Gitpod](https://docs.ddev.com/en/stable/users/install/ddev-installation/#ddev-installation-gitpod) and [GitHub Codespaces](https://docs.ddev.com/en/stable/users/install/ddev-installation/#ddev-installation-codespaces).
```